content/applications/hr/fleet/new_vehicle.rst

```diff
@@ -0,0 +1,146 @@
+   application. When creating a new driver, the driver is added to the *Contacts* application, not
+   the *Employees* application.
+
+Vehicle
+-------
+
+- :guilabel:`Immatriculation Date`: Select the date the vehicle is acquired using the drop-down
+  calendar module.
+- :guilabel:`Cancellation Date`: Select the date the vehicle lease will expire, or when the vehicle
+  will be no longer available, using the drop-down calendar module.
+- :guilabel:`Chasis Number`: Enter the chasis number in the field. This is known in some countries
+  as the :abbr:`VIN (Vehicle Identification Number)` number.
+- :guilabel:`Last Odometer`: Enter the last known odometer reading in the number field. Using the
+  drop-down menu next to the number field, select whether the odometer reading is in kilometers
+  :guilabel:`(km)` or miles :guilabel:`(mi)`.
+- :guilabel:`Fleet Manager`: Select the fleet manager from the drop-down menu, or type in a new
+  fleet manager and click either :guilabel:`Create` or :guilabel:`Create and Edit`.
+- :guilabel:`Location`: Type in the location for the vehicle in the field. The most common scenario
+  for when this field would be populated is if a company has several office locations. The typical
+  office location where the vehicle is located would be the location entered.
+- :guilabel:`Company`: Select the company that the vehicle will be used for and associated with from
+  the drop-down menu, or type in a new company and click either :guilabel:`Create` or
+  :guilabel:`Create and Edit`.
+
+.. image:: new_vehicle/new-vehicle-type.png
+   :align: center
+   :alt: The new vehicle form, showing the vehicle tax section.
+
+Tax Info tab
+------------
+
+Fiscality
+~~~~~~~~~
+
+- :guilabel:`Horsepower Taxation`: Enter the amount that is taxed based on the size of the vehicles
+  engine. This is determined by local taxes and regulations, and varies depending on the location.
+  It is recommended to check with the accounting department to ensure this value is correct.
+- :guilabel:`Disallowed Expense Rate`: Enter the amount of non-deductible expenses for the vehicle.
+  This amount is not counted towards any deductions on a tax return or as an allowable expense
+  when calculating taxable income. It is recommended to check with the accounting department to
+  ensure this value is correct.
+- :guilabel:`Start Date`: Enter the date these two values will affect the taxes, using the
+  drop-down calendar module.
+
+Contract
+~~~~~~~~
+
+- :guilabel:`First Contract Date`: Select the start date for the vehicle's first contract using
+  the drop-down calendar module. Typically this is the day the vehicle is purchased or leased.
+- :guilabel:`Catalog Value (VAT Incl.)`: Enter the MSRP (Manufacturer's Suggested Retail Price) for
+  the vehicle at the time of purchase or lease.
+- :guilabel:`Purchase Value`: Enter the purchase price or the value of the lease for the vehicle.
+- :guilabel:`Residual Value`: Enter the current value of the vehicle.
```

Comment:
Do any of these values impact accounting?

Reply:
I assume so- but these are no "suggested" numbers- they should be concrete- meaning, they do not have any say in what these numbers are. The values are based on formulas (the residual value is) but I am not sure the exact formula. I'll add a note to check with accounting for more info.